daemons/execd/execd_commands.c

```diff
@@ -1555,14 +1555,16 @@ process_lrmd_signon(pcmk__client_t *client, xmlNode *request, int call_id,
                 remoted_request_cib_schema_files();
             }
         } else {
-            rc = -EACCES;
+            rc = EACCES;
         }
 #else
-        rc = -EPROTONOSUPPORT;
+        rc = EPROTONOSUPPORT;
 #endif
     }
 
-    *reply = create_lrmd_reply(__func__, rc, call_id);
+    CRM_LOG_ASSERT(reply != NULL);
```

Comment:
If we're going to do this at all, then we need to either `pcmk__assert()` or `CRM_CHECK()` and return. `CRM_LOG_ASSERT()` just logs and continues to dereference a null pointer.